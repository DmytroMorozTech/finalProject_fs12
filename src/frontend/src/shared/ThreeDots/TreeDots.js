import Style from './styles'
import MoreHorizIcon from '@material-ui/icons/MoreHoriz'
import React from 'react'

function ThreeDots () {
  const classes = Style()
  return (
    <div className={classes.dots}>
      <MoreHorizIcon/>
    </div>
  )
}

export default ThreeDots