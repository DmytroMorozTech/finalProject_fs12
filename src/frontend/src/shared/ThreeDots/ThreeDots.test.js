import React from 'react'
import TreeDots from './TreeDots'
import { shallow } from 'enzyme'

const setUp = () => shallow(<TreeDots />)

describe('should render TreeDots component', () => {
  let component
  beforeEach(() => {
    component = setUp()
  })
  it('to match snapshot', () => {
    expect(component).toMatchSnapshot()
  })
})
