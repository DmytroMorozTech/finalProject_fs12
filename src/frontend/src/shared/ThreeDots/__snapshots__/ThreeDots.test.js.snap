// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render TreeDots component to match snapshot 1`] = `
<Memo(ForwardRef(MoreHorizIcon))
  className="makeStyles-icon-1"
/>
`;
